game/nintendo_mii/gen1_wii_mii.ksy

```diff
@@ -0,0 +1,204 @@
+meta:
+  id: gen1_wii_mii
+  file-extension:
+    - mii
+    - mae
+    - miigx
+    - rcd
+    - rsd
+  endian: be
+seq:
+  - id: invalid
+    type: b1
+    doc: Has no apparent effect.
+  - id: gender
+    type: b1
+    doc: Mii gender. 0 = male, 1 = female.
+  - id: birth_month
+    type: b4
+    doc: Mii birthday month, Ranges from 1 to 12
+  - id: birth_day
+    type: b5
+    doc: Mii birthday day, Ranges from 1 to 31
+  - id: favorite_color
+    type: b4
+    doc: Favorite color. Ranges from 0 to 11.
+  - id: favorite
+    type: b1
+    doc: Whether the Mii is a favorite or not.
+  - id: mii_name
+    type: str
+    size: 20
+    doc: Mii name. Can be up to 10 characters long.
+    encoding: utf-16
+  - id: body_height
+    type: u1
+    doc: Body height. Ranges from 0 to 127, short to tall.
+  - id: body_weight
+    type: u1
+    doc: Body weight. Ranges from 0 to 127, small to large.
+  - id: mii_id
+    type: u1
+    repeat: expr
+    repeat-expr: 4
+    doc: Unique Mii identifier. Also governs color of Mii's pants
+  - id: console_id
+    type: u1
+    repeat: expr
+    repeat-expr: 4
+    doc: Unique identifier for the console the Mii was created on. Consists of a checksum of the first 3 bytes of the console's mac address, followed by the last 3 bytes of the console's mac address
+  - id: face_type
+    type: b3
+    doc: Face shape. Ranges from 0 to 11. Not ordered the same as visible in editor.
+  - id: face_color
+    type: b3
+    doc: Skin color. Ranges from 0 to 9. Not ordered the same as visible in editor. A map of the internal values in correlation to the Mii editor is at /maps.txt/{skin}.
+  - id: facial_feature
+    type: b4
+    doc: Facial feature. Ranges from 0 to 11.
+  - id: unknown
+    type: b3
+    doc: Currently unknown data.
+  - id: no_mingle
+    type: b1
+    doc: Whether the Mii was allowed to travel to other consoles via WiiConnect24. 0 = yes, 1 = no.
+  - id: unknown_2
+    type: b1
+    doc: Currently unknown data.
+  - id: downloaded
+    type: b1
+    doc: Whether the Mii was downloaded from the Check Mii Out channel.
+  - id: hair_type
+    type: b7
+    doc: Hair type. Ranges from 0 to 71. Not ordered the same as visible in editor.
```

Comment:
Should it be a `enum`?

Reply:
I would make it an enum, if I had a copy of the original lookup table, but it's missing from [the repo I extracted these from](https://github.com/RiiConnect24/mii2studio).

Reply:
[I found them!](https://github.com/kaitai-io/kaitai_struct_formats/pull/355#issuecomment-728658246)

This thread (#355) brings up issues about the idea that I don't want to deal with though. 😬